ui/components/app/permission-cell/permission-cell-status.js

```diff
@@ -49,7 +49,7 @@ export const PermissionCellStatus = ({
 
   const renderAccountsGroup = () => (
     <>
-      {process.env.CHAIN_PERMISSIONS ? (
+      {networks.length > 0 ? (
```

Comment:
This confuses me, are we stopping rendering `accounts`? If so, can we remove the code for it?

Reply:
We are just keeping it for Snaps. networks is defined only in case of `permittedChains`.

Reply:
Seems like it is in use for non-Snap connections too. I think this deserves a refactor _outside of this PR_. It feels confusing to nest the `networks` display logic inside the account group logic.

<img width="377" alt="Screenshot 2024-10-10 at 11 49 17" src="https://github.com/user-attachments/assets/fbd9a91f-5285-4b33-8a11-d39e50db4c21">

Reply:
For initial connection screen, we need to show the current active account and all the enabled networks to be there. This is connections screen, it doesn't use permission cell anyway